lib/msf/core/payload/linux/bind_tcp.rb

```diff
@@ -99,10 +95,9 @@ def asm_bind_tcp(opts={})
         mov ecx,esp
         mov al,0x66                   ; socketcall syscall
         int 0x80                      ; invoke socketcall (SYS_SOCKET)
-    ^
 
-    unless close_socket
-      asm << %Q^
+        ; TODO: verify that this is wanted (I think it should be),
+        ; TODO: and look to optimise this a little.
```

Comment:
While doing some testing with the bind payloads I found that the `SO_REUSEADDR` feature wasn't used, which caused reuse of the bind to fail in many cases. I added this in the previous PR, but added this TODO here so that we could properly discuss its inclusion.